dev-env/run-docker-compose.sh

```diff
@@ -49,6 +97,13 @@ if [[ "${1:-""}" == 'restart' ]]; then
   exit
 fi
 
+if [[ "${1:-""}" == 'stop' ]]; then
```

Comment:
The commands `down` and `stop` have a different meaning for `compose`.

- `down` stops the services and remove the containers
- `stop` only stops the services

Here the script if answering to a `stop` positional argument with a `down`, which can be confusing.

As we have already a way to recreate the container with the argument `restart` (the `restart` triggers a `docker compose up --force-recreate`), I suggest to simply call `docker compose stop` and rename the related function accordingly.